src/main/java/org/carlmontrobotics/robotcode2023/RobotContainer.java

```diff
@@ -4,47 +4,67 @@
 
 package org.carlmontrobotics.robotcode2023;
 
+import org.carlmontrobotics.robotcode2023.Constants.OI.Driver;
+import org.carlmontrobotics.robotcode2023.commands.IntakeRoller;
+import org.carlmontrobotics.robotcode2023.commands.OuttakeRoller;
+import org.carlmontrobotics.robotcode2023.subsystems.Roller;
+import org.carlmontrobotics.robotcode2023.subsystems.Roller.RollerMode;
+
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.PowerDistribution;
-import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 
 public class RobotContainer {
 
   public final Joystick driverController = new Joystick(0);
   public final Joystick manipulatorController = new Joystick(1);
   public final PowerDistribution pd = new PowerDistribution();
 
+  public final Roller roller = new Roller();
+
   public RobotContainer() {
     configureButtonBindingsDriver();
     configureButtonBindingsManipulator();
   }
 
-  private void configureButtonBindingsDriver() {}
+  private void configureButtonBindingsDriver() {
+    new JoystickButton(driverController, Driver.rollerIntakeConeButton)
+      .onTrue(new IntakeRoller(roller, RollerMode.INTAKE_CONE, Constants.Roller.conePickupColor));
+    new JoystickButton(driverController, Driver.rollerIntakeCubeButton)
+      .onTrue(new IntakeRoller(roller, RollerMode.INTAKE_CUBE, Constants.Roller.cubePickupColor));
+    new JoystickButton(driverController, Driver.rollerOuttakeConeButton)
+      .onFalse(new OuttakeRoller(roller, RollerMode.OUTTAKE_CONE, Constants.Roller.conePickupColor));
+    new JoystickButton(driverController, Driver.rollerOuttakeCubeButton)
+      .onFalse(new OuttakeRoller(roller, RollerMode.OUTTAKE_CUBE, Constants.Roller.cubePickupColor));
+    new JoystickButton(driverController, Driver.rollerStopButton).onTrue(new InstantCommand(() -> roller.setSpeed(0)));
+  }
   private void configureButtonBindingsManipulator() {}
 
   public Command getAutonomousCommand() {
     return Commands.print("No autonomous command configured");
   }
 
-  private double getStickValue(Joystick stick, XboxController.Axis axis) {
```

Comment:
Doesn't drivetrain need this?

Reply:
Yep, no idea why that got commented out. I've restored it.